recipe/meta.yaml

```
@@ -69,9 +70,15 @@ outputs:
         - libcublas
       run_constrained:
         - arm-variant * {{ arm_variant_type }}  # [aarch64]
-        - {{ pin_subpackage("libcudss-commlayer-nccl", exact=True) }}
-        - {{ pin_subpackage("libcudss-commlayer-mpi", exact=True) }}
+        - libcudss-commlayer-nccl ={{ version }}=*_{{ build_num }}  # [linux]
+        - libcudss-commlayer-mpi ={{ version }}=*_{{ build_num }}*   # [linux]
+        # Cannot pin exactly due to a potentially different build order
+        #- {{ pin_subpackage("libcudss-commlayer-nccl", exact=True) }}
+        #- {{ pin_subpackage("libcudss-commlayer-mpi", exact=True) }}
```

```suggestion
        - {{ pin_subpackage("libcudss-commlayer-nccl", exact=True) }}
        - {{ pin_subpackage("libcudss-commlayer-mpi", exact=True) }}
```
ALSO duplicate these lines into the `requirements.host` of this output.

---

@carterbox I think I've applied the suggested changes (returned pin_subpackage here and added to the host requirement) but the Windows build now fails.